pyproject.toml

```
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of base32-lib
+# Copyright (C) 2019 CERN.
+# Copyright (C) 2019 Northwestern University,
+#                    Galter Health Sciences Library & Learning Center.
+# Copyright (C) 2025 Front Matter.
+
+# base32-lib is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+[build-system]
+requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "base32-lib"
+dynamic = ["version"]
+description = "Small library to generate, encode and decode random base32 strings."
+readme = { file = "README.rst", content-type = "text/x-rst" }
```

v minor:

```suggestion
readme = "README.rst"
```

has been good enough so far if we want to keep it minimalist